bin/webpack-watch-server.js

```diff
@@ -6,6 +6,8 @@ const Liftoff = require('liftoff')
 const chalk = require('chalk')
 const { spawn } = require('child_process')
 
+var argv = require('minimist')(process.argv.slice(2))
```

Comment:
Can the `require` please be broken out to the top of the file with the others, and the `var argv` be replaced with `const args`, just before the `const WebpackWatchServer =`.